rmgpy/cantherm/kinetics.py

```diff
@@ -43,6 +43,7 @@
 
 import rmgpy.quantity as quantity
 import rmgpy.constants as constants
+from rmgpy.molecule.draw import MoleculeDrawer, createNewSurface
```

Comment:
I am a little confused by the functionality of this. Is drawing the PES optional? If it is, should we describe that in documentation? If not, how much time is consumed by this operation relative to calculating the TST equation? When processing many reactions, are separate PES files created or just one with everything drawn on it (which I have seen cantherm do before)?

Reply:
It's fairly fast, so I included it by default (will not run if Cairo isn't installed, though).
I ran two `Kinetics` calcs including PES draw and SA for one of them in just under 10 seconds on my PC.
I just verified that it saves different plots for each PES.